libs/application/templates/aosh/seminars/src/fields/Participants/index.tsx

```
@@ -0,0 +1,280 @@
+        csvInputError.map((csvError: CSVError) => {
+          let messageString = `${formatMessage(
+            participantMessages.labels.csvErrorLabel,
+          )} ${csvError.itemIndex + 1}`
+          csvError.errorList.forEach((errorString) => {
+            messageString = messageString.concat(
+              ' ',
+              formatMessage(errorString),
+            )
+          })
+          return <AlertMessage type="error" message={messageString} />
+        })}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)